project/Dependencies.scala

```diff
@@ -144,6 +147,15 @@ object Dependencies {
     "org.mongodb" %% "casbah" % "3.0.0"
   )
 
+  val cromwellApiClientDependencies = List(
+    "com.typesafe.akka" %% "akka-actor" % cromwellApiClientAkkaV,
+    "com.typesafe.akka" %% "akka-http" % cromwellApiClientAkkaHttpV,
+    "com.typesafe.akka" %% "akka-http-spray-json" % cromwellApiClientAkkaHttpV,
+    "com.github.pathikrit" %% "better-files" % "3.0.0",
+    //---------- Test libraries -------------------//
```

Comment:
Is this picture representing "that's weird"?

This is copy/pasted from the centaur list (one of yours? 😄) - want me to remove it?